torchbenchmark/score/compute_score_v1.py

```diff
@@ -0,0 +1,193 @@
+"""
+Compute the benchmark score given a frozen score configuration and current benchmark data.
```

Comment:
Ideally we wouldn't need 2 files... but looking at it, I don't see a better solution so I guess 2 files is a fine solution for now :)

Reply:
Using two files give us flexibility to design completely different score algorithms. For example, V1 obsoletes the spec file in V0 because now each model has a "task" field so we can extract the domain hierarchy at runtime. It also makes easier to maintain and deprecate the old score versions.